tools/infrasys/src/main.rs

```diff
@@ -0,0 +1,767 @@
+use async_trait::async_trait;
+use log::{debug, info, trace, warn};
+use pubsys_config::{InfraConfig, KMSKeyConfig, SigningKeyConfig};
+use rusoto_cloudformation::*;
```

Comment:
nit: Let's scope this down to the just the things we need. same with the `rusoto_s3`, `std::io::prelude::*` import below

Reply:
For the prelude import, I think ::* might be only option if I'm understanding this page correctly? https://doc.rust-lang.org/std/io/prelude/index.html
The method that uses whatever is imported by prelude is read_to_end (which I use to read a file into a buffer)